src/v2/guide/components-registration.md

```diff
@@ -122,20 +122,20 @@ export default {
 }
 ```
 
-Note that in ES2015+, placing a variable name like `ComponentA` inside an object is shorthand for `ComponentA: ComponentA`, meaning the name of the variable is both:
+Note que en ES2015+, colocar un nombre de variable como `ComponentA` dentro de un objeto es la abreviatura de `ComponentA: ComponentA`, lo que significa que el nombre de la variable es ambos:
 
-- the custom element name to use in the template, and
-- the name of the variable containing the component options
+- el nombre del elemento personalizado para usar en la plantilla, y
+- el nombre de la variable que contiene las opciones de componentes
 
-## Module Systems
+## Sistemas de módulos
 
-If you're not using a module system with `import`/`require`, you can probably skip this section for now. If you are, we have some special instructions and tips just for you.
+Si no estás usando un sistema de módulos con `importar` o `requerir`, probablemente puedes saltarte esta sección por ahora. Si lo estás, tenemos algunas instrucciones y consejos especiales sólo para ti.
```

Comment:
Same here (treat the reader as `usted`). Also, I think we should leave `import` and `require` in english, what do you guys think @sxcamacho @FLarra ?
```suggestion
Si no está usando un sistema de módulos con `import` o `require`, probablemente pueda saltearse esta sección por ahora. Si lo está, tenemos algunas instrucciones y consejos especiales solo para ti.
```

(`solo` is accepted without the accent and I think we are not using accent almost everywhere -- another PR is accepted to normalize this)